omnigibson/systems/micro_particle_system.py

```diff
@@ -1406,6 +1436,7 @@ def _create_particle_prototypes(cls):
         # Load the particle template
         particle_template = cls._create_particle_template()
         og.sim.import_object(obj=particle_template, register=False)
+        cls._particle_template = particle_template
```

Comment:
should _create_particle_template() automatically store the particle template?

Reply:
hmm I am not sure. I feel like it's cleaner if _create_particle_template just call the constructor of a template object, without worrying about importing or registering